habitat-baselines/habitat_baselines/config/default_structured_configs.py

```diff
@@ -47,6 +49,9 @@ class EvalConfig(HabitatBaselinesBaseConfig):
         # available options are "disk" and "tensorboard"
         default_factory=list
     )
+    extra_sim_sensors: Dict[str, SimulatorSensorConfig] = field(
```

Comment:
This is meant to replace `video_render_views`. 
`extra_sim_sensors` contains habitat-lab `SimulatorSensorConfig` definitions that will be added at runtime **only when evaluating**